scripts/deleteDBData.js

```diff
@@ -0,0 +1,28 @@
+import assert from "assert";
+import fetch from "node-fetch";
+import mongoose from "mongoose";
+import User from "../server/api/user/user.js";
+import Chat from "../server/api/chat/chat.js";
+
+const deleteDBData = async () => {
+  let conn;
+  try {
+    conn = await mongoose.connect(
+      "mongodb+srv://admin:admin@arabrew-chat-cluster.kp8fvwt.mongodb.net/?retryWrites=true&w=majority",
+      {
+        useNewUrlParser: true,
+        useUnifiedTopology: true,
+      }
+    );
+    await User.deleteOne({ name: "Benny" });
```

Comment:
what if we have real data with theses names? give a unique ID to the inserted data and use them